lib/customer_io_config.dart

```diff
@@ -0,0 +1,47 @@
+import 'customer_io_enums.dart';
+
+/// Configure plugin using CustomerIOConfig
+class CustomerIOConfig {
```

Comment:
Ah, bummer. Looks like we might need to keep a Dart equivalent to the native SDK `CustomerIOConfig` maintained? 

Sounds like a lot of work that every time that we add a value to the Android or iOS SDK config object, we need to update it across RN SDK and Flutter SDK. But, I am not sure if there is a way around this?

Reply:
Yup, that's the only way, we have to keep these wrapper updated with native SDK